app/src/main/java/com/kickstarter/viewmodels/ChangeEmailViewModel.kt

```diff
@@ -38,6 +42,15 @@ interface ChangeEmailViewModel {
         /** Emits a string to display when email update fails.  */
         fun error(): Observable<String>
 
+        /** Emits a boolean to determine if the user is a creator. */
+        fun isCreator(): Observable<Boolean>
+
+        /** Emits a boolean to determine if a user's email is deliverable. */
+        fun isDeliverable(): Observable<Boolean>
```

Comment:
Same here, if it's not deliverable, what does that mean for the activity that subscribes.